core/new-gui/src/app/workspace/component/property-editor/operator-property-edit-frame/operator-property-edit-frame.component.ts

```diff
@@ -147,58 +147,60 @@ export class OperatorPropertyEditFrameComponent implements OnInit, OnChanges {
   /**
    * Changes the property editor to use the new operator data.
    * Sets all the data needed by the json schema form and displays the form.
-   * @param operatorId
    */
-  showOperatorPropertyEditor(operatorId: string): void {
-    const operator = this.workflowActionService
-      .getTexeraGraph()
-      .getOperator(operatorId);
-    // set the operator data needed
-    this.currentOperatorId = operatorId;
-    const currentOperatorSchema = this.dynamicSchemaService.getDynamicSchema(
-      this.currentOperatorId
-    );
-    this.setFormlyFormBinding(currentOperatorSchema.jsonSchema);
-    this.formTitle = currentOperatorSchema.additionalMetadata.userFriendlyName;
-
-    /**
-     * Important: make a deep copy of the initial property data object.
-     * Prevent the form directly changes the value in the texera graph without going through workflow action service.
-     */
-    this.formData = cloneDeep(operator.operatorProperties);
-
-    // use ajv to initialize the default value to data according to schema, see https://ajv.js.org/#assigning-defaults
-    // WorkflowUtil service also makes sure that the default values are filled in when operator is added from the UI
-    // However, we perform an addition check for the following reasons:
-    // 1. the operator might be added not directly from the UI, which violates the precondition
-    // 2. the schema might change, which specifies a new default value
-    // 3. formly doesn't emit change event when it fills in default value, causing an inconsistency between component and service
-
-    this.ajv.validate(currentOperatorSchema, this.formData);
-
-    // manually trigger a form change event because default value might be filled in
-    this.onFormChanges(this.formData);
-
-    if (
-      this.workflowActionService
+  rerenderEditorForm() {
+    if (this.currentOperatorId) {
```

Comment:
change to if not return

Reply:
this change has been kept since it is only changing the name